### ISSUE TITLE
Recv failure: Connection was reset

### ISSUE THREAD

**Issue by pagnax** [appstract/laravel-opcache#67]
I found the following problem when running command `php artisan opcache:optimize`

Optimize started, this can take a while...
Recv failure: Connection was reset
Url: http://laracon.hcm.com/opcache-api/optimize?key=eyJpdiI6Im1cL0R6VHBTTXMyM1FCd0FObUtsbUZnPT0iLCJ2YWx1ZSI6IlM4XC8wSk01aHFuWnE0aVVBMnhmOWlBPT0iLCJtYWMiOiI2MjE0MDU4NjgyMjQ5YTE3ZDZlNzA5YmZmYTc0MjY5MGIwZjY4YzJkNGU1MThkZTVjYjAxODFiZWI2Zjk4YzE1In0%3D

Please help me with this failure.

Thank.

**Comment by clanofartisans** [comment]
I also ran into this issue today. When I saw that someone else had reported an issue about it, I spent a while trying to track down the issue and I think I've found the problem, at least in my specific case.

For me, the scripts were running into an issue when trying to optimize the `/vendor/composer/autoload_classmap.php` file, specifically when it tried to run `opcache_compile_file()` on it. Not sure if this is a PHP bug or a platform bug.

To work around this issue, I was able to come up with two different ways of getting things running. Only one is necessary so I'm just including both options for the sake of being as thorough as possible.
1. I editted my server's php.ini to add an `opcache.blacklist_filename` directive pointing to a blacklist file. Inside the blacklist file, I added the full path to the offending `autoload_classmap.php` file. I then reloaded my server and my scripts seemed to be running correctly. This is a less than ideal solution as not everyone may have access to edit their server's php.ini file.
2. I published the package's config file (`php artisan vendor:publish --provider OpcacheServiceProvider`). I then simply commented out line 14, `base_path('vendor/composer'),` which prevented the package from attempting to cache files in the composer directory. This is a less than ideal solution as it forces the package to skip attempting to cache the other 6 files in the composer directory.

It _is_ possible this could be a PHP/platform bug as the server I was working with at the time is a Windows x64 server running Apache 2.4 and PHP 7.1 (thread safe). I'm not able to do much more testing at the moment to determine if there's a real correlation here or not. Part of my suspicion stems from the fact that opcache seems to have a few quirks on this server; primarily the `opcache_get_status()` function not reporting `interned_strings_usage` (potentially due to PHP being thread safe, see [this not-necessarily-related issue](https://github.com/amnuts/opcache-gui/issues/26) at [amnuts/opcache-gui](https://github.com/amnuts/opcache-gui/)) as well as some odd handling of wildcards in the blacklist file I had set up for workaround 1. It could also have something to do with the way Windows uses different formatting for describing directory structures compared to *nix.

As the package doesn't appear to be consistently causing widespread issues for others, it sounds like this issue has limited scope so I'm not sure what to recommend to the package maintainer aside from maybe making note of it somewhere in the readme. And, maybe more digging and testing could uncover the true source of the problem.

I hope this helps @pagnax!
